x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.test.tsx

```diff
@@ -14,7 +14,7 @@ import { TimelineEventsDetailsItem } from '../../../../common/search_strategy';
 import { useRuleWithFallback } from '../../../detections/containers/detection_engine/rules/use_rule_with_fallback';
 
 import { TestProviders, TestProvidersComponent } from '../../mock';
-import { TimelineId } from '../../../../common';
+import { TimelineId } from '../../../../common/types';
```

Comment:
can this be a `import type`?

Reply:
These can, but most of the times these are auto-inferred by typescript. There are linter rules for forcing these with an auto-fix as well so I am not going to overly worry about them in these few cases for this PR.